resource/bem/index.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html>
+
+<head>
+    <title> Prechu </title>
+    <link rel="stylesheet" type="text/css" href="../bem/css.css" />
+</head>
+
+<body>
+    <div class="page">
+        <div class="page__header">
+            <div class="multi-columns">
+                <div class="page__header__left">
```

Comment:
it seems we have problem with two-columns component. Please contact others for help